src/main/java/eu/execom/pomodoroTeam/controllers/LoginController.java

```diff
@@ -34,22 +43,48 @@ public Principal user(Principal principal) {
         return principal;
     }
 
-    @GetMapping("/user/getAll")
-    public Iterable<UserEntity> getAllUsers() {
-        return userRepository.findAll();
-
+    @GetMapping("/getAll")
+    public ResponseEntity<List<UserEntity>> getAllUsers() {
+        return new ResponseEntity<>(userRepository.findAll(), HttpStatus.OK);

```

Comment:
You should not chain method calls, but rather assign users to a variable, and then pass it to a constructor.

Reply:
I am not sure how to handle this. If i need return list, i have to put list as a ResponseEntity type.